ur_robot_driver/src/robot_state_helper_node.cpp

```diff
@@ -35,6 +36,8 @@ int main(int argc, char** argv)
   ros::init(argc, argv, "ur_robot_state_helper");
   ros::NodeHandle nh;
 
+  registerUrclLogHandler();
+
```

Comment:
This node should not need that. It is plain ROS (except for using datatypes from the library)

Reply:
I will remove the loghandler from this node again.